code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -154,6 +154,20 @@
 			new /obj/item/storage/belt/chameleon(src) // Unique but worth at least 2 tc
 			new /obj/item/card/id/syndicate(src) // 2 tc
 			new /obj/item/chameleon(src) // 7 tc
+			
+		if("ancient") //A kit so old, it's probably older than you. //This bundle is filled with the entire unlink contents traitors had access to in 2006, from OpenSS13. Notably the esword was not a choice but existed in code.
+			/obj/item/storage/toolbox/emergency/old/ancientbundle //Items fit neatly into a classic toolbox just to remind you what the theme is.
+		
+/obj/item/storage/toolbox/emergency/old/ancientbundle/ //So the subtype works
```

Comment:
```suggestion
new /obj/item/storage/toolbox/emergency/old/ancientbundle(src) //So the subtype works
```

A fix from upstream

Reply:
i'm pretty sure the new was supposed to go to line 159 and not 161.